File: shared/src/commonMain/kotlin/com/medhelp/callmedcmm2/model/ScheduleTomorrowResponse.kt
package com.medhelp.callmedcmm2.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class ScheduleTomorrowResponse {
    @SerialName("error")
    var error = false
    @SerialName("message")
    var message: String? = null
    @SerialName("response")
    var response: List<ScheduleTomorrowItem> = ArrayList<ScheduleTomorrowItem>()

    @Serializable
    class ScheduleTomorrowItem {
        @SerialName("data")
        var data: String? = null
        @SerialName("start")
        var start: String? = null
        @SerialName("end")
        var end: String? = null
        @SerialName("kab")
        var kab: String? = null
        @SerialName("naim_filial")
        var naim_filial: String? = null
    }
}

File: iosApp/iosApp/data/model/ScheduleTomorrowResponse.swift
//
//  ScheduleTomorrowResponse.swift
//  CallMed_ios
//
//  Created by Mihail on 04.05.2023.
//

import Foundation

struct ScheduleTomorrowResponse: Decodable,Encodable{
    var error : Bool?
    var message : String?
    var response : [ScheduleTomorrowItem]?
}
